components/rumi_ai/actions/create-session/create-session.mjs

```diff
@@ -0,0 +1,121 @@
+import app from "../../rumi_ai.app.mjs";
+
+export default {
+  key: "rumi_ai-create-session",
+  name: "Create Session",
+  description: "Create a new session/meeting. [See the documentation](https://rumiai.notion.site/Rumi-Public-API-Authentication-02055b7286874bd7b355862f1abe48d9)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    sessionTitle: {
+      propDefinition: [
+        app,
+        "sessionTitle",
+      ],
+    },
+    about: {
+      description: "A brief description of the session.",
+      propDefinition: [
+        app,
+        "about",
+      ],
+    },
+    startDate: {
+      propDefinition: [
+        app,
+        "startDate",
+      ],
+    },
+    startTime: {
+      propDefinition: [
+        app,
+        "startTime",
+      ],
+    },
+    endDate: {
+      propDefinition: [
+        app,
+        "endDate",
+      ],
+    },
+    endTime: {
+      propDefinition: [
+        app,
+        "endTime",
+      ],
+    },
+    enableRecording: {
+      propDefinition: [
+        app,
+        "enableRecording",
+      ],
+    },
+    summaAI: {
+      propDefinition: [
+        app,
+        "summaAI",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const {
+      app,
+      sessionTitle,
+      about,
+      startDate,
+      startTime,
+      endDate,
+      endTime,
```

Comment:
Is there a reason to use separate props for date and time in this app? We normally do date-time strings in a single prop